vnext/PropertySheets/External/Microsoft.ReactNative.WindowsSdk.Default.props

```diff
@@ -5,13 +5,19 @@
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <ImportGroup Condition="'$(ReactExperimentalFeaturesSet)' != 'true'">
```

Comment:
Does this need to be here? If the import is missing in the playground project, can you just add it there?

Reply:
When you import React.Cpp.props, it imports Microsoft.ReactNative.WindowsSdk.Default.props, and then later JSEngine.props.  Currently JSEngine.props is the one that imports ExperimentalFeatures.props.

Maybe I should move all the logic around ReactExperimentalFeaturesSet directly into React.cpp.props?

Reply:
Up to now, the logic has been:

1. App/module projects load ExperimentalFeatures.props directly because it's part of the template
2. The M.RN projects have never loaded ExperimentalFeatures.props directly, they only get them when built as dependencies by an app/module (which did load the props)
3. JSEngine.props, have those extra props loaded in, (which maybe means they get loaded eventually in M.RN projects?)

The M.RN projects do load React.Cpp.props.

Since they're supposed to be global, I think it makes sense that the logic gets moved into React.Cpp.props, but it might cascade and have a larger effect on its own (since then things like test projects will start loading it). Let me spin off a separate PR with just that change.